kafka-avro-schema-quickstart/src/test/java/org/acme/kafka/KafkaAndSchemaRegistryTestResource.java

```diff
@@ -0,0 +1,40 @@
+package org.acme.kafka;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.strimzi.StrimziKafkaContainer;
+
+public class KafkaAndSchemaRegistryTestResource implements QuarkusTestResourceLifecycleManager {
+
+    private final StrimziKafkaContainer kafka = new StrimziKafkaContainer();
+
+    private GenericContainer<?> registry;
+
+    @Override
+    public Map<String, String> start() {
+        kafka.start();
+        registry = new GenericContainer<>("apicurio/apicurio-registry-mem:1.3.2.Final")
+                .withExposedPorts(8080)
+                .withEnv("QUARKUS_PROFILE", "prod")
+                .withEnv("KAFKA_BOOTSTRAP_SERVERS", kafka.getBootstrapServers())
```

Comment:
the env vars `KAFKA_BOOTSTRAP_SERVERS` , `APPLICATION_ID` and `APPLICATION_SERVER` are not required, the image `apicurio/apicurio-registry-mem:1.3.2.Final` stores the schemas in-memory and do not require access to kafka

Reply:
Good point. I'm updating the quickstart to use Apicurio Registry 2.0.0, so I'll remove this as part of the work.